src/pudl/etl/cli.py

```diff
@@ -37,24 +36,12 @@ def pudl_etl_job_factory(
         The job definition to be executed.
     """
 
-    def get_pudl_etl_job():
+    def get_pudl_etl_job(job_name: str | None = None):
         """Create an pudl_etl_job wrapped by to be wrapped by reconstructable."""
         pudl.logging_helpers.configure_root_logger(logfile=logfile, loglevel=loglevel)
-        jobs = [define_asset_job("etl_job")]
-        if not process_epacems:
-            jobs = [
-                define_asset_job(
-                    "etl_job",
-                    selection=pudl.etl.create_non_cems_selection(
-                        pudl.etl.default_assets
-                    ),
-                )
-            ]
-        return Definitions(
-            assets=pudl.etl.default_assets,
-            resources=pudl.etl.default_resources,
-            jobs=jobs,
-        ).get_job_def("etl_job")
```

Comment:
Oh I see... we redefined the definitions here but didn't add the asset checks so they weren't getting run. Importing [`defs` from `pudl.etl`](https://github.com/catalyst-cooperative/pudl/blob/23612af209a58c382fca7b424596c90563b36b1a/src/pudl/etl/__init__.py#L266-L269) includes the assets checks.